intl/en/sl-deploy.txt

```diff
@@ -0,0 +1,64 @@
+usage: {{%MAIN% [options] [URL [PACK|BRANCH]]}}
+
+Deploy a node application to a StrongLoop process manager
+
+Options:
+  {{-h,--help}}          Print this message and exit.
+  {{-v,--version}}       Print version and exit.
+  {{-s,--service SVC}}   Deploy to service SVC.
```

Comment:
sure.  Eventually, our Tech. Editor is willing to review all the *.txt.  He and i went over the {placeholder} and {{hard-code}} notation, etc.